src/ContainerMigrateModal.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Modal } from 'patternfly-react';
+import { Button, Checkbox, Select, SelectOption } from '@patternfly/react-core';
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+const targetHostPlaceholder = _("Choose target host...");
```

Comment:
That makes sense, I'll move it to the class. (The primary purpose of the placeholder being a constant was to avoid a typo/mismatch breaking the comparison in the `Migrate` button.)